website/docs/concepts/router.md

```diff
@@ -36,60 +44,14 @@ browser's current URL and passes it to the `render` callback. The callback then
 In case no path is matched, the router navigates to the path with `not_found` attribute. If no route is specified,
 nothing is rendered, and a message is logged to console stating that no route was matched.
 
-```rust
-use yew_router::prelude::*;;
-use yew::prelude::*;
-
-#[derive(Clone, Routable, PartialEq)]
-enum Route {
-    #[at("/")]
-    Home,
-    #[at("/secure")]
-    Secure,
-    #[not_found]
-    #[at("/404")]
-    NotFound,
-}
-
-#[function_component(Secure)]
-fn secure() -> Html {
-    let history = use_history().unwrap();
-
-    let onclick_callback = Callback::from(move |_| history.push(Route::Home));
-    html! {
-        <div>
-            <h1>{ "Secure" }</h1>
-            <button onclick={onclick_callback}>{ "Go Home" }</button>
-        </div>
-    }
-}
-
-fn switch(routes: &Route) -> Html {
-    match routes {
-        Route::Home => html! { <h1>{ "Home" }</h1> },
-        Route::Secure => html! {
-            <Secure />
-        },
-        Route::NotFound => html! { <h1>{ "404" }</h1> },
-    }
-}
-
-#[function_component(Main)]
-fn app() -> Html {
-    html! {
-        <Switch<Route> render={Switch::render(switch)} />
-    }
-}
-```
-
```

Comment:
This huge example is repeated right below minus the context. It panics on app start.